released/discovery_center/mission_3239/README.md

```diff
@@ -0,0 +1,56 @@
+# Discovery Center mission - Setup for SAP S/4HANA side-by-side UI Extensions on SAP Business Technology Platform
+## Overview
+
+This sample shows how to setup your SAP BTP account for the Discovery Center Mission - [Setup for SAP S/4HANA side-by-side UI Extensions on SAP Business Technology Platform](https://discovery-center.cloud.sap/missiondetail/3239)
+
+## Content of setup
+
+The setup comprises the following resources:
+
+- Creation of the SAP BTP subaccount
+- Entitlements of services
+   * SAP Business Application Studio
+   * SAP Build Work Zone, standard edition
+   * Continous Integration & Delivery - optional
+   * Integration of Custom Identity & Authentication Service - optional
+  
+- Subscriptions to applications
+- Role collection assignments to users
+
+## Deploying the resources
+
+Make sure that you are familiar with SAP BTP and know both the [Get Started with btp-terraform-samples](https://github.com/SAP-samples/btp-terraform-samples/blob/main/GET_STARTED.md) and the [Get Started with the Terraform Provider for BTP](https://developers.sap.com/tutorials/btp-terraform-get-started.html)
+
+To deploy the resources you must:
+
+1. Create a file `secret.auto.tfvars` and maintain the credentials for the BTP provider
+
+   ```hcl
```

Comment:
I would consider this as best practice: it should at least be mentioned that the user and password can be provided via environment variables